RadarSDK/RadarTimeZone.m

```diff
@@ -56,7 +67,7 @@ - (NSDictionary *)dictionaryValue {
     dict[@"_id"] = self._id;
     dict[@"name"] = self.name;
     dict[@"code"] = self.code;
-    dict[@"currentTime"] = self.currentTime;
+    dict[@"currentTime"] = [timezoneDateFormatter() stringFromDate:self.currentTime];
```

Comment:
the current time string from server is a different format (no millisecond) so we need a different parser than RadarUtils.isoDateFormatter